reference/component-versions.md

```diff
@@ -5,6 +5,67 @@ This documents the versions of the different components for each Anbox Cloud rel
 
 Not all components are updated with each release. When components are not updated, they are marked with `n/a` below.
 
+## 1.23.0
+
+### Charms
+
+#### Ubuntu 22.04
+
+| Name | Channel | Revision |
+|----------|--------------|--------------|
+| `anbox-cloud-dashboard` | `1.23/stable` | ? |
+| `ams-node-controller` (Deprecated) | `1.23/stable` | ? |
+| `coturn` | `1.23/stable` | ? |
+| `ams` | `1.23/stable` | ? |
+| `aar` | `1.23/stable` | ? |
+| `anbox-stream-gateway` | `1.23/stable` | ? |
+| `ams-lxd` | `1.23/stable` | ? |
+| `anbox-stream-agent` | `1.23/stable` | ? |
+| `nats ` | `latest/stable` | 11 |
+
+#### Ubuntu 20.04 (Deprecated)
+
+| Name | Channel | Revision |
+|----------|--------------|--------------|
+| `anbox-cloud-dashboard` | `1.23/stable` | ? |
+| `ams-node-controller` (Deprecated) | `1.23/stable` | ? |
+| `coturn` | `1.23/stable` | ? |
+| `ams` | `1.23/stable` | ? |
+| `aar` | `1.23/stable` | ? |
+| `anbox-stream-gateway` | `1.23/stable` | ? |
+| `ams-lxd` | `1.23/stable` | ? |
+| `anbox-stream-agent` | `1.23/stable` | ? |
+| `nats ` | `latest/stable` | 11 |
+
+### Bundles
+
+| Name | Channel | Revision |
+|----------|--------------|--------------|
+| `anbox-cloud` | `1.23/stable` | ? |
+| `anbox-cloud-core` | `1.23/stable` | ? |
+
+### Snaps
+
+| Name |  Channel | Version |
+|----------|--------------|---------|
+| `ams`    | `1.23/stable` | 1.23.0 |
+| `aar`    | `1.23/stable` | 1.23.0 |
+| `ams-node-controller` (Deprecated) | `1.23/stable` | 1.23.0 |
+| `anbox-cloud-dashboard` | `1.23/stable` | 1.23.0 |
+| `anbox-stream-agent` | `1.23/stable` | 1.23.0 |
+| `anbox-stream-gateway` | `1.23/stable` | 1.23.0 |
+
+### Anbox images
```

Comment:
We didn't had this before but technically we have each image for VM and container. Should we duplicate entries, ignore this or do something else?

Reply:
Since the image names are technically the same, we don't have to duplicate. But I have added a line to indicate we have a variant for each : VM and container